source/unified-test-format/unified-test-format.rst

```diff
@@ -2635,22 +2635,34 @@ sections.
 Terminating Open Transactions
 `````````````````````````````
 
-Open transactions can cause tests to block indiscriminately. Test runners SHOULD
-terminate all open transactions at the start of a test suite and after each
-failed test by killing all sessions in the cluster. Using the internal
-MongoClient, execute the ``killAllSessions`` command on either the primary or,
-if connected to a sharded cluster, all mongos servers.
+Open transactions can cause tests to block indiscriminately. When connected to
+MongoDB 3.6 or later, test runners SHOULD terminate all open transactions at the
+start of a test suite and after each failed test by killing all sessions in the
+cluster. Using the internal MongoClient, execute the ``killAllSessions`` command
+on either the primary or, if connected to a sharded cluster, all mongos servers.
 
 For example::
 
     db.adminCommand({
       killAllSessions: []
     });
 
-The test runner MAY ignore any command failure with error Interrupted(11601) to
-work around `SERVER-38335`_.
+The test runner MAY ignore the following command failures:
+
+- Interrupted(11601) to work around `SERVER-38335`_.
+- Unauthorized(13) to work around `SERVER-54216`_.
+- CommandNotFound(59) if the command is executed on a pre-3.6 server
```

Comment:
Just being flexible in case a test runner does decide to call `killAllSessions` regardless of server version.